src/index.css

```diff
@@ -1,13 +1,72 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+:root {
+  --red-500: #960035;
+  --red-400: #c50035;
+  --red-300: #d71818;
+  --red-200: #ff002a;
+  --red-100: #f66060;
+  --grey-400: #121212;
+  --grey-300: #212121;
+  --grey-200: #707070;
+  --grey-100: #c4c4c4;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.text-red-500 {
+  color: var(--red-500);
+}
+
+.text-red-400 {
+  color: var(--red-400);
+}
+
+.text-red-300 {
+  color: var(--red-300);
+}
+
+.text-red-200 {
+  color: var(--red-200);
+}
+
+.text-red-100 {
+  color: var(--red-100);
+}
+
+.text-grey-100 {
+  color: #707070;
+}
+
+.text-grey-200 {
+  color: var(--grey-200);
+}
+
+.text-grey-300 {
+  color: var(--grey-300)
+}
+.text-grey-400 {
+  color: var(--grey-400);
+}
+
+.bg-grey-300 {
+  background-color: var(--grey-300);
+}
+
+.bg-darker-400 {
+  background-color: var(--grey-400);
+}
```

Comment:
If we're using tailwind, we don't need these changes here to the CSS

If we must add a brand specific color that is not included in TailwindCSS, we can do so by adding it in the config